events/api.py

```diff
@@ -1714,7 +1713,8 @@ def _filter_event_queryset(queryset, params, srs=None):  # noqa: C901
         queryset = (
             queryset.filter(**kwargs)
             .filter(
-                end_time__gte=datetime.utcnow().replace(tzinfo=pytz.utc), deleted=False
+                end_time__gte=datetime.utcnow().replace(tzinfo=ZoneInfo("UTC")),
```

Comment:
SonarCloud complains about:

Don't use `datetime.datetime.utcnow` to create this datetime object.

So better use:
```
from datetime import datetime, timezone

datetime.now(timezone.utc)
```